Bam92/wayFarer
# #167090255 Show all available trips

#### What does this PR do?
Display all available trips 
#### Description of Task to be completed?
A click on **View all trips** button displays all trips offered by the app
#### How should this be manually tested?
Visit the home page of the app
Scroll down till you see **View all trips** button
Click that button
#### Any background context you want to provide?
This implementation does not allow user to hide some trips
#### What are the relevant pivotal tracker stories?
#167090255
#### Screenshots (if appropriate)
![view-all-trips](https://user-images.githubusercontent.com/14941675/60823400-bcc2bc80-a1a7-11e9-9756-1fa5127989fd.png)
